modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -1,126 +1,132 @@
-* The migration script supports MySQL, Postgres, Oracle, and Microsoft SQL Server. There is no migration for h2 database.
+=== Custom Portal Look & Feel
+*For target version older than Bonita 2021.2 only*: there is no guarantee that the Portal Look & Feel definition is compatible across maintenance versions.
+For example, in Bonita 6.2.2, `jquery+` was renamed `jqueryplus` in `BonitaConsole.html`, for compatibility with more application servers. +
+If you are using a custom Look & Feel for Bonita Portal, export it before launching the update procedure.
+After the update is complete, export the default Look & Feel from the new version, recreate your custom Look & Feel from the new default one and with the recommendations for form footers. Then, import your updated custom Look & Feel into Bonita Portal.
 
 [WARNING]
 ====
 
-*Important:* +
-The migration operation resets the Bonita configuration files to default version for new settings to be applied (from the _$BONITA_HOME_ folder in <7.3.0 version or inside database in >=7.3.0).
-Therefore, you must do a  xref:bonita-bpm-platform-setup.adoc#update_platform_conf[backup of your configuration files] before starting the migration. +
-You will need to merge custom properties and configurations to the migrated environment.
-
-Furthermore, from the database point of view, as any operations on a production system, a migration is not a zero risk operation. +
-Therefore, it is strongly recommended to do a  xref:back-up-bonita-bpm-platform.adoc[backup of your database] before starting the migration.
+Starting with Bonita 2021.2, there is no more Bonita Portal. Bonita is all applications. If you need to use some of the Portal Look&Feel assets in the themes of your applications, make sure you create backups of those files before launching the updating procedure.
 ====
 
-== Estimate time required
+=== Custom configuration
+
+To update Bonita to a *target version Bonita 7.3 or greater*: Starting with Bonita 7.3, there is no more _bonita home_ folder. +
+What this means:
+
+* If you have customized your configuration, you will have to use the xref:bonita-bpm-platform-setup.adoc#update_platform_conf[setup tool] to send your customized configuration files to the database where the configuration is stored
+* If your current installation does not have any custom configuration, then you do not need to configure the bundle any further
+
+== Estimated time required
 
-The platform must be shut down during migration.
+Bonita Runtime must be shut down during the update. +
 The time required depends on several factors including the database volume, the number of versions between the source version and the
-target version, and the system configuration,
-so it is not possible to be precise about the time that will be required. However, the following example can be used as a guide:
+target version, and the system configuration, so it is not possible to be precise about the time that will be required. +
```

Comment:
about the time that will be required. + => about the required time. +

Reply:
ok